### PR TITLE
If page is PDP, exclude current product from Recently Viewed display …

### PR DESCRIPTION
**Pull request by SysRivarian**
…due to redundancy.

### CODE REVIEW

**Comment by philwinkle** [comment]
Apologies for radio silence. Reviewing these PRs now.

File: skin/frontend/base/default/js/sd_recentlyviewed/recent.js [hunk 1]
```diff
@@ -62,13 +62,23 @@ SomethingDigitalRecentlyViewed.prototype = {
 		var items = this.getLoadedStorage();
 
 		return items.map(function(a){
+			if($(document.body).hasClassName('catalog-product-view')){
+
+				var currentProductId = $$('#product_addtocart_form input[name=product]')[0].value;
```
Comment (philwinkle) [inline skin/frontend/base/default/js/sd_recentlyviewed/recent.js:67]: This likely should be configurable (like the other html selectors) from with in the admin to maintain theme compatibility.

Comment (SysRivarian) [inline skin/frontend/base/default/js/sd_recentlyviewed/recent.js:67]: Agreed (and no worries :)).

**Comment by philwinkle** [comment]
Closed for inactivity. Please feel free to reopen.

**Pull request closed without merging**